float64 z_err
float64[6] ee_vel
float64 force_z
float64 force_des_z